src/components/LeaderboardRanking.vue

```diff
@@ -13,9 +15,9 @@
         <div v-for="user in users" :key="user.id" class="d-flex">
           <div class="user-avatar text-center flex-shrink-0">
             <div class="relative rounded-full overflow-hidden">
-              <cld-context v-if="user.photoKey" :cloudName="cloudName">
+              <cld-context v-if="user.photoKey || user.photo_key" :cloudName="cloudName">
                 <div class="">
-                  <cld-image :publicId="user.photoKey">
+                  <cld-image :publicId="user.photoKey || user.photo_key">
```

Comment:
I think it would be preferable to fix this on the BE directly to see why the key is not formatted correctly for the current user.